src/metadataparsers/mods/CdmToMods.php

```diff
@@ -233,7 +234,7 @@ public function createPageLevelModsXML($page_pointer, $page_title, $xmlSnippet =
         if ($includeMigratedFromUri == true) {
             $CONTENTdmItemUrl = '<identifier type="uri" invalid="yes" ';
             $CONTENTdmItemUrl .= 'displayLabel="Migrated From">';
-            $CONTENTdmItemUrl .= 'http://content.lib.sfu.ca/cdm/ref/collection/';
+            $CONTENTdmItemUrl .= 'http://cdm16313.contentdm.oclc.org/cdm/ref/collection/';
             $CONTENTdmItemUrl .= $collectionAlias . '/id/'. $page_pointer .'</identifier>';
             $modsOpeningTag .= $CONTENTdmItemUrl;
         }
```

Comment:
We've got an [open issue](https://github.com/MarcusBarnes/mik/issues/176) to remove this logic entirely and implement the migrate from URL functionality as a metadata manipulator.